app/models/contest.rb

```diff
@@ -20,6 +20,16 @@ def started?
     start + signup_duration < Time.current
   end
 
+  def submit_metrics?(contest_owner_id)
```

Comment:
methods ended with `?` should return Boolean.
https://stackoverflow.com/questions/7179016/what-is-the-purpose-of-and-at-the-end-of-method-names